.flowconfig

```diff
@@ -40,6 +40,10 @@ node_modules/@zulip/shared/
 [libs]
 node_modules/react-native/Libraries/react-native/react-native-interface.js
 node_modules/react-native/flow/
+; If the following line is not present, Flow ignores this file in favor of
+; flow-typed/npm/@sentry/react-native_vx.x.x.js. (Contrast commit 6219e5a23,
+; where the opposite behavior is exhibited!)
+flow-typed/@sentry/react-native_v1.x.x.js
```

Comment:
Huh! Peculiar. I have no good hypotheses to explain this after a couple of minutes' looking; I might poke at it a bit more later.

Reply:
It is concerning. In particular I worry that it's only pseudodeterministic – based on the ordering of keys in a hashmap, or what-have-you. 

(I spent some time hunting through the source myself before writing this hack, but no luck. :sweat:)